pkg/tests/tasks/traffic/request_routing_test.go

```diff
@@ -40,41 +45,70 @@ func TestRequestRouting(t *testing.T) {
 		app.InstallAndWaitReady(t, app.Bookinfo(ns))
 
 		productpageURL := app.BookinfoProductPageURL(t, meshNamespace)
-		testUserCookieJar := app.BookinfoLogin(t, meshNamespace)
+		// testUserCookieJar := app.BookinfoLogin(t, meshNamespace)
 
 		t.NewSubTest("not-logged-in").Run(func(t TestHelper) {
 			oc.ApplyString(t, ns, app.BookinfoVirtualServicesAllV1)
 
+			// String value of the Review pod name
+			reviewV1Podname := strings.TrimSpace(shell.Execute(t, fmt.Sprintf(`oc get pods -n %s | grep reviews-v1 | awk '{print $1}'`, ns)))
```

Comment:
You can use `pod.MatchingSelector` to get podLocator and inside you can use pod.Name to get the name of the pod

Reply:
will use this one